.travis.yml

```diff
@@ -2,4 +2,30 @@ language: java
 sudo: false
```

Comment:
BTW, a clarification that I'm putting here for posterity:  it *seems* that a new release will be created whenever a new tag is created, since the `.travis.yml` file specifies:  `deploy.on.tags: true`.  i.e., so it seems to me that it's not true that the releases will be created "only" when we create a new release on the https://github.com/mesos/storm/releases page.